### PR TITLE
Run PHP CS Fixer in CI

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+    cs:
+        name: 'Code Style'
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga:2.18.5
```
Comment (OskarStark) [inline .github/workflows/ci.yml]:
```suggestion
              uses: docker://oskarstark/php-cs-fixer-ga:2.18.6
```

?

I tagged the release a minute ago

Comment (derrabus) [reply .github/workflows/ci.yml]: Oh good, I was about to ask you for a bump. 😅

Comment (derrabus) [reply .github/workflows/ci.yml]:
```
Pull down action image 'oskarstark/php-cs-fixer-ga:2.18.6'
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 1.367 seconds before retry.
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 6.592 seconds before retry.
  /usr/bin/docker pull oskarstark/php-cs-fixer-ga:2.18.6
  Error response from daemon: manifest for oskarstark/php-cs-fixer-ga:2.18.6 not found: manifest unknown: manifest unknown
Error: Docker pull failed with exit code 1
```

Comment (OskarStark) [reply .github/workflows/ci.yml]: The tag was not created, can you retry?

Comment (derrabus) [reply .github/workflows/ci.yml]: Looks good now, thanks. 🙂

**Review by OskarStark — APPROVED** [review]

**Pull request merged**